### ISSUE TITLE
Dark Mode support

### ISSUE THREAD

**Issue by sriharshachilakapati** [AntonPoltoratskyi/NativeUI#5]
I'm using this to show progress indicator dialog, and while it is looking great in regular mode, it is not showing up properly in dark mode. Any idea how to do that?

![Screen Shot 2021-05-04 at 3 57 41 PM](https://user-images.githubusercontent.com/4593204/116990980-81d13780-acf1-11eb-99e0-0499b4230eb7.png)

Also, would it be possible to remove the separator and actions area completely? I'll be dismissing it programmatically once network call is done.

**Comment by devpolant** [comment]
Hello @sriharshachilakapati! In case if you need to display some loading indicator without any actions at the bottom - you probably need to use another UI component like this https://github.com/jdg/MBProgressHUD or any other `Progress HUD` alternative. 

> Would it be possible to remove the separator and actions area completely?

Currently it is not supported. Can't promise, but maybe I can add this closer to the end of the week.

**Comment by sriharshachilakapati** [comment]
That's a pretty quick reply! Thanks!!

I'm new to the iOS development scene and am looking for this alternative for a course project which I've to complete by tomorrow. I think I'll go with MBProgressHUD as you recommended.

Out of curiosity, I've went into the Pods directory and edited the background color property to UIColor.systemBackgroundColor and still it showed me light mode UI. Any idea what went wrong there? I'd like to try myself how that works after this project so I can learn.

**Comment by devpolant** [comment]
@sriharshachilakapati It looks light enough because of these lines of code in `AlertView.swift`:
```swift
    private lazy var blurView: UIVisualEffectView = {
        let blurEffect = UIBlurEffect(style: .extraLight)
        
        let effectView = UIVisualEffectView(effect: blurEffect)
        effectView.translatesAutoresizingMaskIntoConstraints = false
        addSubview(effectView)
        
        return effectView
    }()
```

So background is not solid color, but blur actually.

**Comment by sriharshachilakapati** [comment]
So setting the style there to `.systemMaterial` should fix it right? Please allow me to make a PR for this one. Will try once I submit this project for grading.

**Comment by devpolant** [comment]
@sriharshachilakapati not sure if it will fix it. Need more investigation before doing any change and merge. The main purpose of this pod was to implement the alert which will look exactly like the system one.

But if you need to make changes temporarily - you can make a fork -> push commit -> specify your fork repo and branch in podfile (see https://stackoverflow.com/a/39930762)

**Comment by devpolant** [comment]
@sriharshachilakapati Both issues are fixed:
- dark mode fixed here https://github.com/devpolant/NativeUI/pull/7
- action buttons can be empty since now https://github.com/devpolant/NativeUI/pull/6

I published new version - `1.2.1` which includes this.
